tests/phpunit/elementor/core/files/css/test-base.php

```diff
@@ -0,0 +1,91 @@
+<?php
+namespace Elementor\Tests\Phpunit\Elementor\Core\Files\Css;
+
+use ElementorEditorTesting\Elementor_Test_Base;
+
+/**
+ * Test the CSS Base class
+ */
+class Test_Base extends Elementor_Test_Base {
+
+	/**
+	 * @var \Elementor\Core\Files\CSS\Post
+	 */
+	private static $css_base_class;
+	/**
+	 * @var array
+	 */
+	private static $mock_control;
+	/**
+	 * @var array[]
+	 */
+	private static $mock_controls_stack;
+
+	public static function setUpBeforeClass() {
+		// The CSS Base class is abstract, so it can't be instantiated. The inheriting Post class is used instead.
+		self::$css_base_class = new \Elementor\Core\Files\CSS\Post( 0 );
+
+		self::$mock_control = [
+			'type' => 'number',
+			'default' => 20,
+		];
+
+		self::$mock_controls_stack = [
+			'number' => self::$mock_control,
+		];
```

Comment:
same as before, it's better for future-devs readability to have all of the "arrange" part in the test itself instead of having to look for it. In this case it's not a big deal, but when this file will have tons of tests it will be

Reply:
I changed the method to the non-static `setUp()` instead of `setUpBeforeClass()`.
When more tests will be added to this file, this can be refactored to an "arrange helper" function instead of `setUp()` so it doesn't run for unnecessary tests. But for now I think this is fine.